addons/product/models/product_template.py

```diff
@@ -830,12 +830,12 @@ def _get_own_attribute_exclusions(self):
         product_template_attribute_values = self.valid_product_template_attribute_line_ids.product_template_value_ids
         return {
             ptav.id: [
-                value_id
+                value_id.id
                 for filter_line in ptav.exclude_for.filtered(
                     lambda filter_line: filter_line.product_tmpl_id == self
-                ) for value_id in filter_line.value_ids.ids
+                ) for value_id in filter_line.value_ids if value_id.ptav_active
```

Comment:
```suggestion
                ) for value in filter_line.value_ids if value.ptav_active
```
variable doesn't hold an id anymore, so its name should be updated as well ;)